src/python/serve_html.py

```diff
@@ -61,7 +61,7 @@
                     "pin": 3
                 }
             ],
-            "serial-protocol": 3,
+            "serial-protocol": 4,
```

Comment:
Not required, I assume this was just for your testing.

Reply:
done